### ISSUE TITLE
Is there a way to mandate <template> before <script> in a file?

### ISSUE THREAD

**Issue by jnardone** [vuejs/eslint-plugin-vue#1235]
**Please describe what the rule should do:**
We would like to be able to style check that developers always have their <template> section before their <script> section.

**What category should the rule belong to?**
- [x] Enforces code style
- [ ] Warns about a potential error
- [ ] Suggests an alternate way of doing something
- [ ] Other (please specify:)

For example, I'd want to be notified about this:

```vue
<script>
export default {
  name: 'test',
  ...
}
</script>

<template>
  <div>
    <v-flex />
  </div>
</template>
```

**Comment by loren138** [comment]
I think you're looking for https://eslint.vuejs.org/rules/component-tags-order.html#vue-component-tags-order which is new in v7 (which is currently alpha).

**Comment by jnardone** [comment]
yes, perfect!
